### ISSUE TITLE
perf: many jan-web error logs when restarting jan services

### ISSUE THREAD

**Issue by 0xSage** [janhq/jan#46]
To repro: 
1. Docker compose up
2. ctrl+c, stop all
3. docker compose up again

It works but for ~1 minute I get the scary logs: 

```sh
jan-web-1                   | Error refreshing access token TypeError: fetch failed
jan-web-1                   |     at Object.fetch (node:internal/deps/undici/undici:11576:11)
jan-web-1                   |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
jan-web-1                   |   cause: AggregateError
jan-web-1                   |       at internalConnectMultiple (node:net:1114:18)
jan-web-1                   |       at afterConnectMultiple (node:net:1667:5)
jan-web-1                   |       at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17) {
jan-web-1                   |     code: 'ECONNREFUSED',
jan-web-1                   |     [errors]: [ [Error], [Error] ]
jan-web-1                   |   }
jan-web-1                   | }
jan-web-1                   | Error refreshing access token TypeError: fetch failed
jan-web-1                   |     at Object.fetch (node:internal/deps/undici/undici:11576:11)
jan-web-1                   |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
jan-web-1                   |   cause: AggregateError
jan-web-1                   |       at internalConnectMultiple (node:net:1114:18)
jan-web-1                   |       at afterConnectMultiple (node:net:1667:5)
jan-web-1                   |       at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17) {
jan-web-1                   |     code: 'ECONNREFUSED',
jan-web-1                   |     [errors]: [ [Error], [Error] ]
jan-web-1                   |   }
jan-web-1                   | }
jan-web-1                   | Error refreshing access token TypeError: fetch failed
jan-web-1                   |     at Object.fetch (node:internal/deps/undici/undici:11576:11)
jan-web-1                   |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
jan-web-1                   |   cause: AggregateError
jan-web-1                   |       at internalConnectMultiple (node:net:1114:18)
jan-web-1                   |       at afterConnectMultiple (node:net:1667:5)
jan-web-1                   |       at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17) {
jan-web-1                   |     code: 'ECONNREFUSED',
jan-web-1                   |     [errors]: [ [Error], [Error] ]
jan-web-1                   |   }
jan-web-1                   | }
jan-web-1                   | Error refreshing access token TypeError: fetch failed
jan-web-1                   |     at Object.fetch (node:internal/deps/undici/undici:11576:11)
jan-web-1                   |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
jan-web-1                   |   cause: AggregateError
jan-web-1                   |       at internalConnectMultiple (node:net:1114:18)
jan-web-1                   |       at afterConnectMultiple (node:net:1667:5)
jan-web-1                   |       at TCPConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17) {
jan-web-1                   |     code: 'ECONNREFUSED',
jan-web-1                   |     [errors]: [ [Error], [Error] ]
jan-web-1                   |   }
jan-web-1                   | }
jan-web-1                   | - wait compiling /api/auth/logout/route (client and server)...
jan-web-1                   | - event compiled successfully in 181 ms (1812 modules)
jan-web-1                   | token is expired, refresh it
jan-web-1                   | token is expired, refresh it
jan-web-1                   | token is expired, refresh it
jan-web-1                   | token is expired, refresh it
jan-web-1                   | token is expired, refresh it
jan-web-1                   | token is expired, refresh it
```

**Comment by louis-jan** [comment]
the issue I mentioned this morning, jan frontend instance is spin up before other (graphql-engine, keycloak...) cc @hiento09 @namchuai. Service dependencies watch could work.

**Comment by namchuai** [comment]
Re-assign to @louis-jan since it's been fixed with PR: #60
